lib/pluginmanager/templates/codec-plugin/logstash-codec-example.gemspec.erb

```diff
@@ -2,9 +2,9 @@ Gem::Specification.new do |s|
   s.name          = 'logstash-codec-<%= plugin_name %>'
   s.version       = '0.1.0'
   s.licenses      = ['Apache-2.0']
-  s.summary       = 'TODO: Write a short summary, because Rubygems requires one.'
-  s.description   = 'TODO: Write a longer description or delete this line.'
-  s.homepage      = 'TODO: Put your plugin''s website or public repo URL here.'
+  s.summary       = 'Logstash Codec Plugin for <%= classify(plugin_name) %>'
```

Comment:
if left as a `TODO`, `bundle install` fails with:
```
$ bundle install
You have one or more invalid gemspecs that need to be fixed.
The gemspec at /Users/seanstory/Desktop/Dev/logstash-input-xkcd/logstash-input-xkcd.gemspec is not valid. Please fix this gemspec.
The validation error was '"FIXME" or "TODO" is not a description'
```

In order to prevent the need for manual intervention, this seemed to be a better default - providing a reasonable summary, and commenting out `description` and `homepage`, which are optional.

Reply:
Nice point, we should ensure that those 2 properties are valued  before publishing. This is nice. Do you know how we could enforce that check?

Reply:
No, I do not know a way to enforce that check. And I don't think we should enforce it, since rubygems does not require those to be provided. I think its helpful for us to have them here so that they can see them (an IDE will still note these lines as TODOs), but to let them enforce their own standards before they publish the gem. WDYT?

Reply:
The majority of Logstash's plugins has those two field valued, so I think it's a sort of standard it has. Being a gem targeted for Logstash, we should keep this practice and so forcing the developer to populate it. If we remove the `TODO` or remove those 2 settings, I doubt that a developer remembers to fill it.
So my opinion is to make some actions so that a developer remember to do it, could it be keeping the `TODO` or enforcing the existence at gem publish time. I personally prefer the first, so that once done, the path to publish the gem is smooth.

Reply:
👍 ok, I'll remove the comments from these